docs/keps/20200619-federated-resource-status.md

```diff
@@ -282,6 +367,22 @@ By following this approach, kubefed would be able to properly consume and report
 of any federated resource by checking the `status.conditions` (e.g `Ready=True`) fields.
 
 
+**Note that**, this proposed solution might include additional flags to the kubefed control-plane
+components to avoid blowing out the control-plane due to frequent and concurrent API calls to
+update the status of the federated resources.
+
+Obviously users might want to be able to enable/disable the definition of:
+
+* Which condition to look for each federated resource to determine its readiness, e.g. `Ready=True`, `Deployed=True`.
+This might be especially useful for custom resource types without a `IsReady` standard condition.
+
+* When to show up the raw status of the resources or just the failed status (as today).
+By raw status, the system understand to show the status of all the federated resources `Ready` and not `Ready`.
```

Comment:
1. By raw status, I assume its the copy of the status that appears under the status subresource of an arbitraty k8s resource. Atleast that is what I meant when I did refer to it earlier. Do you mean something else here?
2. To compute the `Ready/NotReady` status of the federated resource you would anyways need to fetch and go through the status of each individual resource. If any additional detail, for example the `message` that you have alluded in the example above, you might need to go through/parse all conditions that appear in that subresource. IMHO, this itself incurs a performance overhead. Do you yet have any insight/idea on what/how much this might be?

Reply:
1. Yes, I understood the same by raw status.
2. I was planning to use the ObservedGeneration to exclude out of date conditions. I didn't run an estimation yet, I am waiting to define this kep into implementable stage to proceed defining some solutions and/or limitations. We all agree this approach might be benefitial but there are some performance challenges.

Reply:
@irfanurrehman Do you have any opinions or additional questions here?

Reply:
Fine with me.